setup.cfg

```diff
@@ -82,7 +82,7 @@ plugins =
 
 [coverage:report]
 precision = 2
-fail_under = 97.69
+fail_under = 98.80
```

Comment:
Why I'm seeing this line? Isn't it rebased on master? :thinking:

Reply:
Not sure why this is still in the diff. It's been rebased again, so there shouldn't be any difference here. :/